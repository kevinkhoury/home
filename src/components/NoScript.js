import React from 'react'

export default () => (
  <noscript>
    <div
      style={{
        textAlign: 'center',
        fontSize: '20px',
        padding: '20px',
        margin: 'auto',
      }}
    >
      <h3>Javascript Disabled</h3>
      <p>
        This site requires javascript to display content. Please enable
        javascript and try again.
      </p>
    </div>
  </noscript>
)
