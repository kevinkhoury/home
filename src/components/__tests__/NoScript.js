import React from 'react'
import renderer from 'react-test-renderer'

import NoScript from '../NoScript'

describe('NoScript', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NoScript />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
