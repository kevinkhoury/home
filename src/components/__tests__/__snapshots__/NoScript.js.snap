// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoScript renders correctly 1`] = `
<noscript>
  <div
    style={
      Object {
        "fontSize": "20px",
        "margin": "auto",
        "padding": "20px",
        "textAlign": "center",
      }
    }
  >
    <h3>
      Javascript Disabled
    </h3>
    <p>
      This site requires javascript to display content. Please enable javascript and try again.
    </p>
  </div>
</noscript>
`;
